fillers/eips/eip6780/eip6780.py

```diff
@@ -0,0 +1,138 @@
+"""
+Test EIP-6780: remove selfdestruct
+"""
+from typing import Dict
+
+from ethereum_test_forks import ShanghaiToCancunAtTime15k
+
+from ethereum_test_tools import (
+    Account,
+    CodeGasMeasure,
+    Environment,
+    StateTest,
+    TestAddress,
+    Transaction,
+    Yul,
+    test_only,
+    to_address,
+    to_hash,
+)
+from ethereum_test_tools.vm.opcode import Opcodes as Op
+
+REFERENCE_SPEC_GIT_PATH = "EIPS/eip-6780.md"
+REFERENCE_SPEC_VERSION = "2f8299df31bb8173618901a03a8366a3183479b0"
+
+@test_only(fork=ShanghaiToCancunAtTime15k)
+def test_eip6780_create_selfdestruct_same_tx(fork):
+        env = Environment(
+                coinbase="0x2adc25665018aa1fe0e6bc666dac8fc2697ff9ba",
+                difficulty=0x20000,
+                gas_limit=10000000000,
+                number=1,
+                timestamp=1000,
+        )
+        post: Dict[Any, Any] = {}
+
+        test_ops =[
+                "6032",
+                "601c",
+                "6000",
+                "39", # codecopy
```

Comment:
This bytecode can be replaced with the `Op` library to make it more readable.

For example, these lines can be replaced with `Op.CODECOPY(0, 28, 50)`, which will automatically place the numbers in the stack using PUSH operations.

Reply:
yes, that would be a good improvement.